modules/ROOT/pages/spring-data-neo4j.adoc

```diff
@@ -1,318 +1,478 @@
 = Spring Data Neo4j
+:doctype: book
 :level: Intermediate
 :page-level: Intermediate
+:manualIncludeDir: https://raw.githubusercontent.com/spring-projects/spring-data-neo4j/master/
 :author: Jennifer Reif
 :programming-language: java
 :category: drivers
-:tags: spring, spring-data, sdn, app-development, applications
-:description:  For Java developers who use the Spring Framework or Spring Boot, this guide introduces Spring integration through the Spring Data Neo4j library.
-:neo4j-version: FOO
-
-[NOTE]
-====
-The Spring Data Neo4j 5.x project is currently for existing support and maintenance only.
-All new and future projects should use Spring Data Neo4j 6 project instead.
-This will be part of the upcoming Spring Data Release train Ockham (2020.0.0).
-Please consult the https://github.com/spring-projects/spring-data-commons/wiki/Release-Train-Ockham-(2020.0.0)[wiki] for detailed information about the release dates.
-An updated version of this guide will be released along with the GA.
-====
+:tags: spring, spring-data, SDN, reactive, app-development, applications
+:description: For Java developers who use the Spring Framework or Spring Boot and want to take advantage of reactive development principles, this guide introduces Spring integration through the Spring Data Neo4j project.
+include::{manualIncludeDir}README.adoc[tags=properties]
+// TODO: New release coming in October - this page will be rebuilt
+:neo4j-version: 4.2.0
```

Comment:
should this be in the antora config ? like for the drivers?

Reply:
https://github.com/neo4j-documentation/developer-guides/blob/publish/antora.yml#L13